mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -231,7 +231,7 @@ private int calculateTotalTrackers() {
                 totalTrackers+=trackers.length;
             }
         }
-        return totalTrackers;
+        return totalTrackers + calculateCliqzAttrackCount() + getAdBlockCount() + getSmartBlockingCount();
```

Comment:
I think it's better to have a different function to count them all, and rename this function.